src/components/Header/Header.jsx

```diff
@@ -0,0 +1,96 @@
+import { Switch } from '@mui/material';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import { styled } from '@mui/material/styles';
+
+import Logo from '../../assets/images/NavLogo.png';
+import theme from '../../theme/theme';
+import styles from './Header.module.scss';
+
+// eslint-disable-next-line no-shadow
```

Comment:
It might be a good idea to move `ThemeSwitcher` to a separate file like `styles.js` and import it from there to avoid the name shadowing.